kubevirt-101/intro.md

```diff
@@ -0,0 +1,5 @@
+KubeVirt is a virtual machine management add-on for Kubernetes. The aim is to provide a common ground for virtualization solutions on top of Kubernetes.
+
+This scenario demonstrates how KubeVirt can be used to start a virtual machine
```

Comment:
The sentence looks incomplete maybe? What about finishing it with something like:

> on a Kubernetes cluster using standard Kubernetes mechanisms.

At the very least it needs a closing period.

Reply:
We are already saying so in the paragraph abowe.  The closing period works for me.